### DISCUSSION TITLE
MEETING: etaxonomy Working Group - 13 Apr 2022 09:00 – 10:30

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#281] [MEETINGS]
# Date: 13 Apr @ 09:00 – 10:30
# Agenda: 

* [Standing Agenda items from 29 Mar 2022](https://github.com/fieldmuseum/EMu-Documentation/discussions/277)
* [Mammals questions](https://docs.google.com/document/d/1qgOZM4hISjR3cJIPLemvGtiF2lEluxSvMF0MFZ40-4s)

## Creating Synonymies
* Homotypic/Combination
* Heterotypic/synonym
* [Workflow doc for creating synonymies](https://docs.google.com/document/d/1PnftDxe-oRp1VFrlQsoFSFx796S4xKzPJxdLJDtdSv0/edit)

## Sharing Taxonomy Records
* Action, Botany and Zoology
* resourceRelationship

## Indicating identification certainty (ecatalogue but relates to taxonomy)
* Qualifier - 
* * [Lookup List](https://github.com/fieldmuseum/EMu-Documentation/blob/master/lookuplists/shared/Qualifiers) 
* * [Field Help](https://github.com/fieldmuseum/EMu-Documentation/issues/282)
* Qualifier Rank 
* Confidence field

**Comment by rondlg** [comment]
# MEETING NOTES:
Attendees: AF; NS; PM; WG; TW; MvK; JM; ZK; MT; RdP; RS; KGJ; MT; JJ; KW; SG

## Creating Synonymies

### How to update/create synonomies - refresher

>For EMu-purposes (& not necessarily following ICN/ICZN/etc codes):
- Homotypic Synonymy = Genus is different but specific epithet is the same [?]
- Heterotypic Synonymy = Genus and specific epithet are different [?]

> There is a tool in EMu that lets you set the colours of your text. For example you can make an edited fields turn blue until you save.

### Searching for synonyms
> How do I search for an old name and get the synonyms?
> NS: Would you only get homotypic spp when searching? 
> If you search Genus = XYZ, records where ClaGenus = XYZ come back (no search also, no synonyms)
> To retrieve synonyms, go to All Synonyms tab, & search ‘Synonym name’ (SynNameRef_tab); alt’ly stay on Classification tab & search Current Name (ClaCurrentNameRef) = XYZ …then could use those resulting Taxonomy records to search Catalog for specimens ID’ed as any of those Taxa 
> BUT - this will currently retrieve Catalogue records ‘ever’ identified as any of those taxa. (Check before re-identifying.)

> What they want to do is to search for the old value and get the new ones back as well.
> Also what happens on the web? (For PH)
> MVK : The taxonomy search on botany-collections web search doesn’t seem to be working as expected.
 E.g. Frullania rostrata (Hook. f. & Taylor) Hook. f. & Taylor - https://collections-botany.fieldmuseum.org/taxonomy/93741 
> Add SynNameRef_tab fields to core?

> Difference between what happens in taxonomy and catalogue
> Updating taxonomy records with synonymy doesn’t update any catalogue record data.
> When you need to update catalogue records because of some updated taxonomy you can search for the taxa and then go to the objects tab.

> Label Report: Can I get the currently accepted name printed out on my labels?
 PM: Would like the filed as name in bold: Currently accepted name smaller underneath.

### How bib works with Taxonomy
> Attaching to a correctly formed bib record on the Primary Citation tab will trigger autofilling of authors in the taxonomy record.


## Sharing Taxonomy Records [DRAFT doc started here](https://docs.google.com/document/d/1sVg4GtZG8r5KOKohp7rbVgTJIyZKGSqlovKhrBm2QdA/edit?usp=sharing).
> Not sharing taxonomy records has become a problem.
> **DECISION: Does anyone have a problem with other people using their taxonomy records? Answer = NO!**

**To Do:** 
> Primary owner department retains edit permissions
> Need to take into account/come up with a workflow for the augmentation of taxonomy records with information/media from other departments.
> Workflow/s for attaching images to Taxonomy (+ Narrative for context)

**Check impacts on searching**
> Check if filtering for searching Tax from Catalogue would be affected / or could still be filtered in Registry on certain fields (e.g. filter on > Identification tab fields but Not on resourceRelationship fields.)
> Power Users – consider how they need Mgr vs Users to be able to search/attach/etc

## Creating New taxa
> How do we handle people outside of a collection area creating taxa for them for example for sightings/observations?
> Collections would like to be notified 
> Fill in flat (non-parent) preliminary taxonomy record for a proposed taxon
> Notify the collection so that they can then attach up to the parent and if necessary work out any conflicting hierarchy questions.
> There can be different arrangements by collection
>Mammals is ok for Action to create taxa
>IZ/Insects would like to be informed 

**To Do:** 
> Get a list of agreed taxonomic source/standard lists from which everyone can work.
> Then anyone who needs to can create a taxonomy record from that list without permission from the collection. Whether the parent is attached is then decided between the two groups.
> “Zach and Michelle, if you want to do the list approach I think we could use the mammal diversity database for that purpose: https://www.mammaldiversity.org/ Seems like an easy approach that only highlights odd cases/issue that may arise periodically”

> There are some duplicate Invertebrate records. Can we get a list from RDP to share with PM for decisions and potential merging? 

## CITES
> Need to reopen the conversation on getting CITES data into EMu. For the next WG meeting

## Indicating identification certainty (ecatalogue but relates to taxonomy)
> How is everyone using the Confidence v Qualifiers?
[ See ISSUE#282](https://github.com/fieldmuseum/EMu-Documentation/issues/282)
See [ISSUE #519](https://github.com/fieldmuseum/EMu-Documentation/issues/519) 

> ZK to check with Nancy if/how she is using it?
> Mammals uses if but not much, so happy to change to a different usage.

**Comment by fmjjones** [comment]
Agenda items for next WG meeting:

CITES data and interim plan for importing to Taxonomy. Relates to issue [#503](https://github.com/fieldmuseum/EMu-Documentation/issues/503)

**Comment by zkachian** [comment]
Question: We’re wondering how to handle situations where the taxa in WI and the taxa in EMu do not match? We’d like to avoid a situation where our inventory data end up using different names (for example, if we have a camera trap occurrence that has to use a WI taxon, but we have other observations where we would potentially want to use a more up-to-date taxon that’s in EMu).

**Comment by fmrdelapena** [comment]
Question:  Are we allowed to use brackets "[  ]" instead of parentheses in EMu for authorship? Particularly for use when descriptions were originally published anonymously.

**Reply by fmjjones** [reply]
Formatting of the author string is hard coded based on Taxonomic code value (ICZN vs ICBN) and Author role (e.g., ex Author, MS Author) assigned to each author on the Authors tab. There isn't a role for anonymous authors so my suggestion would be to assign the role of "Author" and use the Automatic? = No value for Scientific Name formation on the Classification tab and manually add the square brackets around the name. For everyone's reference see [Recommendation 51D](https://www.iczn.org/the-code/the-code-online/) of the ICZN.
